lambda_functions/execute_ssh_command_js/serverless.yml

```diff
@@ -1,31 +1,47 @@
 # Welcome to Serverless!
 #
+# This file is the main config file for your service.
+#
 # For full config options, check the docs:
 #    docs.serverless.com
 #
 # Happy Coding!
+service: execute-on
 
-service: execute-ingest  # NOTE: update this with your service name
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+# frameworkVersion: "=X.X.X"
 
+# plugins:
 plugins:
+  # install serverless pseudo parameters using the following:
+  #   1) $ npm install serverless-pseudo-parameters --save-dev
   - serverless-pseudo-parameters
 
+custom:
+  Stage: ${opt:stage, self:provider.stage}
+  CustEnv:
+    s3Bucket:
+      prod: "This is production environment"
+      dev: "dea-lambdas-dev"
+
 provider:
   name: aws
   runtime: nodejs6.10
   environment:
-    DEA_MODULE: dea/20180515
-    PROJECT: v10
-    QUEUE: normal
     hostkey: 'orchestrator.raijin.users.default.host'
     userkey: 'orchestrator.raijin.users.default.user'
     pkey: 'orchestrator.raijin.users.default.pkey'
+    DEA_MODULE: dea/20180515
+    PROJECT: v10
+    QUEUE: express
   region: ap-southeast-2
+  deploymentBucket: ${self:custom.CustEnv.s3Bucket.${self:custom.Stage}}
```

Comment:
Tricky! This is cool. I'll fix up your AWS permissions so we can work out a production bucket to use soon!